src/RadhydroSimulation.hpp

```diff
@@ -606,10 +614,11 @@ template <typename problem_t> void RadhydroSimulation<problem_t>::fillPoissonRhs
 	// NOTE: in the future, this should also deposit particle mass
 	auto const &state = state_new_cc_[lev].const_arrays();
 	auto rhs = rhs_mf.arrays();
+	const Real G = Gconst_;
 
 	amrex::ParallelFor(rhs_mf, [=] AMREX_GPU_DEVICE(int bx, int i, int j, int k) noexcept {
 		// copy density to rhs_mf
-		rhs[bx](i, j, k) = 4.0 * M_PI * state[bx](i, j, k, HydroSystem<problem_t>::density_index);
+		rhs[bx](i, j, k) = 4.0 * M_PI * G * state[bx](i, j, k, HydroSystem<problem_t>::density_index);
```

Comment:
@BenWibking - wouldn't other test(s) that use this function fail now, since the default value of `G` is the CGS value? Earlier, G = 1, so this worked fine.

But no other tests fail - so does that mean no other test use this function? I'm confused

Reply:
I am also confused, since I thought the SphericalCollapse test also uses this.

Reply:
@BenWibking did you find anything here?

Reply:
@markkrumholz this is the remaining issue. Earlier, we had `G = 1` for every test problem. Now, we have `G = 6.67e-8` as the default value The Spherical Collapse test now uses the default value of `G`, so we expected it to fail, but the test still passes, which is why we are confused.

Reply:
I see -- well I guess the question is whether it uses that same value of G to compute the analytic solution against which it is comparing. If the code is run using the same value of G as is used to compute the analytic solution, it should still pass, because the analytic solution is computed consistently with the value used in the simulation. However, in case the pass is trivial because the run time wil be much too short; changing G from 1 to 6.67e-8 increases the free-fall time by a factor of 1/sqrt(6.67e-8), so if the run time was left the same, then the effect is that instead of running for ~1 free-fall time, the test now runs of ~10^-4 of a free-fall time. So the solution is simply to increase the run time by a factor of 1/sqrt(6.67e-8), and the test should then be doing exactly what it was doing before.

Reply:
We don't compute a reference solution for this test. We just check if the exit status is 0 at the end of the run. I have now increased the simulation run time by 1/sqrt(6.67e-8), as the freefall time has indeed increased by this factor.

Reply:
Ah, ok. I forgot we didn't add a reference solution for that test. For the purposes of the SphericalCollapse test only, I think it would make sense to just set G = 1.

Reply:
We can, but there's no reason not to have it take on its CGS value. The problem is completely self-similar, so the unit system is arbitrary. Time just gets measured in units of the free-fall time.